var/www/skitter.fanfic.dev/style.css

```diff
@@ -0,0 +1,26 @@
+body { 
+  margin: auto;
+  padding: 1em;
+  max-width: 960px; 
+  font-size: 1.2em;
+  font-family: 'Karla', sans-serif;
+  line-height: 2rem;
+}
+
+body h2 { 
```

Comment:
There isn't an `h2` in your current markup

Reply:
Deleted